wg-node-lifecycle/charter.md

```diff
@@ -0,0 +1,202 @@
+- Allow the kubelet to trigger rescheduling of pods: https://docs.google.com/document/d/1-wJhiNy84w7tzFdo9HqwTu5DrVSuXFLGTUv8FBiRAAc
+- Implicit tolerations https://github.com/kubernetes/enhancements/issues/5282; add a Filter plugin to ensure that non-GPU pods are not scheduled on GPU nodes: https://github.com/kubernetes-sigs/scheduler-plugins/pull/812
+- Node Resource Hot Plug: https://github.com/kubernetes/enhancements/issues/3953
+
+## Relevant Projects
+
+This is a list of known projects that solve similar problems in the ecosystem or would benefit from
+the efforts of this WG:
+
+- https://github.com/aws/aws-node-termination-handler
+- https://github.com/foriequal0/pod-graceful-drain
+- https://github.com/jukie/karpenter-deprovision-controller
+- https://github.com/kubereboot/kured
+- https://github.com/kubernetes/autoscaler/tree/master/cluster-autoscaler
+- https://github.com/kubernetes-sigs/cluster-api/
+- https://github.com/kubernetes-sigs/karpenter
+- https://github.com/kubernetes-sigs/kubespray
+- https://github.com/kubevirt/kubevirt
+- https://github.com/medik8s/node-maintenance-operator
+- https://github.com/Mellanox/maintenance-operator
+- https://github.com/NVIDIA/pika
+- https://github.com/openshift/machine-config-operator
+- https://github.com/planetlabs/draino
+- https://github.com/strimzi/drain-cleaner
+
+There are also internal custom solutions that companies use.
+
+## Prioritization
+
+The group activity will focus on the following features:
+
+- Declarative Node Maintenance
+- EvictionRequest API
+- Graceful Node Shutdown
+
+And have the following priorities which mostly apply to WG calls, but can also apply to the general
+WG review/work/guidance capacity:
+1. Urgent topics concerning the WG focus features during the KEP and code greeze periods.
+2. Discussing issues within the scope of the WG.
+3. Presentations within the scope of the WG.
+4. The WG focus features outside the KEP and code freeze periods.
+5. Other WG features or features within the scope of the WG. If a topic is suggested multiple times,
+   try to prevent starvation.
+
+## Roles and Organization Management
+
+This WG adheres to the Roles and Organization Management outlined in [wg-governance]
+and opts-in to updates and modifications to [wg-governance].
+
+[wg-governance]: /committee-steering/governance/wg-governance.md
+
+## Timelines and Disbanding
+
+The working group will disband once the features and core APIs defined in the following
+KEPs/Features have reached a stable state (GA) and ongoing maintenance ownership is established
+within the relevant SIGs. We will review whether the working group should disband if appropriate
+SIG ownership can't be reached.
+
+We expect to focus on bringing the following features to a stable state (GA):
+- Declarative Node Maintenance
+- EvictionRequest API
+- Graceful Node Shutdown
+
+We will also reevaluate the exit criteria upon the disbandment.
```

Comment:
This is odd statement to put here, I'm not quite sure what you're trying to say here. The exit criteria define the disbandment, and you've clearly defined them above. They are not strict and they will be debatable, but this statement is contradicting that. I'd suggest to entirely drop this.

Reply:
Ack, updated.